red.py

```diff
@@ -97,6 +99,33 @@
     else:
         logger.exception(type(error).__name__, exc_info=error)
 
+async def update_check():
+    if settings.owner == "id_here":
+        print("No owner set!")
+        return
+    owner = discord.utils.get(bot.get_all_members(), settings.owner)
+    try:
+        async with aiohttp.get("https://api.github.com/repos/Twentysix26/Red-DiscordBot/commits") as r:
+            gh_commits = await r.json()
+        latest_sha = gh_commits[0]["sha"][:7]
+        cur_version = os.popen(r'git show -s HEAD --format="%cr|%s|%h"')
```

Comment:
Even though this should block for nearly no time at all, it's still bad practice. Check the `version` command in the Owner cog for a better (async) way to do this.